src/cryptonote_core/cryptonote_core.h

```diff
@@ -33,8 +33,6 @@
 #include <ctime>
 
 #include <boost/function.hpp>
-#include <boost/program_options/options_description.hpp>
```

Comment:
Again, direct references to `progam_options` but no obvious forward declare. Defined indirectly via `command_line.h` - so no obvious benefit to this change.